### ISSUE TITLE
Docker image `v0.2.0` is broken

### ISSUE THREAD

**Issue by zemanlx** [GoogleCloudPlatform/container-structure-test#64]
### What is broken

```bash
docker run -i --rm gcr.io/gcp-runtimes/container-structure-test:v0.2.0 --help
/go_default_test: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```

Probably wrong base image...I am bit confused by your _bazel_ machinery.

### What I expect

Same thing as for `v0.1.3`

```bash
docker run -i --rm gcr.io/gcp-runtimes/container-structure-test:v0.1.3 --help                                                                                                      2s 
Usage of /go_default_test:
  -driver string
        driver to use when running tests (default "docker")
  -image string
        path to test image
  -pull
        force a pull of the image before running tests
  -save
        preserve created containers after test run
  -test.bench regexp
        run only benchmarks matching regexp
  -test.benchmem
        print memory allocations for benchmarks
  -test.benchtime d
        run each benchmark for duration d (default 1s)
...
```

**Comment by dlorenc** [comment]
Hey,

Sorry about this. It was fixed in #61, but we have to cut a release to get a new image out. I'll do one today.

**Comment by zemanlx** [comment]
Fixed in the release [v0.2.1](https://github.com/GoogleCloudPlatform/container-structure-test/releases/tag/v0.2.1). 

Thanks!
